ci/gpu/build.sh

```diff
@@ -242,7 +242,7 @@ else
 fi
 
 if [ -n "\${CODECOV_TOKEN}" ]; then
-    codecov -t \$CODECOV_TOKEN
+    codecov -v -t \$CODECOV_TOKEN
 fi
```

Comment:
This block can be deleted. Our Jenkins job has a codecov upload command in it by default, so it is safe to remove.

The reason you see an error from this is because the escape character is incorrect syntax so you aren't referencing a variable.